hw1/css/style.css

```diff
@@ -0,0 +1,142 @@
+@import 'fonts.css';
```

Comment:
The widget looks good in all browsers, good job!
Just fix the position of the image in IE 
![Screenshot_4](https://user-images.githubusercontent.com/53054365/94304560-6b534880-ff78-11ea-9420-8c33c602bfdd.jpg)